apiserver/facades/client/application/backend.go

```diff
@@ -118,8 +119,36 @@ type stateShim struct {
 	*state.State
 }
 
-type modelShim struct {
-	*state.Model
+type modelInfoShim struct {
```

Comment:
Add a comment that this is just needed for the legacy code to create a provider in the facade and will be removed once al the logic is migrated to the domain service.